tpm2/credactivation/credential_activation_test.go

```diff
@@ -11,6 +11,14 @@ import (
 	"github.com/google/go-tpm/tpm2"
 )
 
+func mustDecodeBase64(in string) []byte {
+	d, err := base64.StdEncoding.DecodeString(in)
+	if err != nil {
+		panic(err)
```

Comment:
pass in `testing.T` and call `t.Fatal` instead

Reply:
Done :)